src/pages/HomePage/HomePage.js

```diff
@@ -43,21 +60,87 @@ const offices = [
   }
 ];
 
+const event = [
+  {
+    title: "Career Fair @ Temple University"
+  },
+  {
+    title: "Resume Review Workshop @ SAC"
+  },
+  {
+    title: "Penn Hackathon @ UPenn"
+  },
+  {
+    title: "Vanguard Networking Event @ SERC"
+  }
+];
+const meeting = [
+  {
+    title: "Today 9am-10am : Sarah Parker"
+  },
+  {
+    title: "Tomorrow 11am-12pm : Jack Doe"
+  },
+  {
+    title: "Next week: Jane Smith"
+  },
+  {
+    title: "Next week: Dr Lasname"
+  }
+];
 export function HomePage() {
-  const user = useContext(UserContext);
-
   const classes = useStyles();
   return (
     <>
-      <h1>Home page</h1>
-      <h2>
-        Current user name is {user.firstName} {user.lastName}
-      </h2>
-      <p>This is the home page</p>
+      <br></br>
+      <div className={classes.page}>
+        <center>
+          <br></br>
+          <h1>Welcome to BetterUni</h1>
+          <img src="images/bulogo.png" width="100" height="100"></img>
+          <h2>We aim to provide you the tools, opportunities,</h2>
+          <h2> and resources to succeed at Temple Univeristy and beyond</h2>
+        </center>
+        <br></br>
+      </div>
+      <br></br>
       <div className={classes.content}>
-        <AdvisingOfficeList offices={offices} />
-        <AdvisingOfficeList offices={offices} />
-        <AdvisingOfficeList offices={offices} />
+        <HomeInfoList
+          avatar={
+            <Avatar
```

Comment:
Could you adjust the import statement for Avatar to be `import {Avatar as ListIcon} from "@material-ui/core";` then just change all instances of `<Avatar ... / >` to `<ListIcon ... />`?